### ISSUE TITLE
jade-mode doesn't support tabs

### ISSUE THREAD

**Issue by strugee** [brianc/jade-mode#66]
The title kinda sums it up.

**Comment by czert** [comment]
Yup. See: https://github.com/brianc/jade-mode/blob/master/sws-mode.el#L138

Seems like I'll have to finally learn elisp and fix this, as it is actually a bit of a problem for me...
